build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -67,10 +39,6 @@
 
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
 
-  <Target Name="LogOutput">
-    <Message Importance="High" Text="%(NuSpec.Identity) -> %(NuSpec.NuPkgOutput)'" />
-  </Target>
-
```

Comment:
Currently we do not. I think changing verbosity to detailed from normal and adding an echo command before this line will help.